public/container-template.json

```
@@ -33,99 +33,90 @@
         "supportZones": true
       },
       "tagIds": [
-        "GTM-TX7J9RB"
+        "GTM-PF23SRX"
```

I made this small change after you copied it.
```suggestion
        "GTM-USEYOUROWN"
```